.github/workflows/docfx-build-and-push.yml

```diff
@@ -13,14 +13,14 @@ on:
       - 'docs/docfx/**'
       - '.github/workflows/dotnet.yml' 
       - '!**/*.md' # Exclude markdown files
-  pull_request:
-    branches: [ "main" ]
-    paths:
-      - 'Kepware.Api/**'
-      - 'Kepware.Api.Sample/**'
-      - 'KepwareSync.Service/**'
-      - '.github/workflows/dotnet.yml'  
-      - '!*.md' # Exclude markdown files
+  # pull_request:
```

Comment:
@rlabbeptc i have changed this as i receive this error:
![image](https://github.com/user-attachments/assets/7b2189be-1d31-4836-9f81-9e3bae52a7ab)

Reply:
That's ok for now. I'm still new to Actions, but my goal was to have it run and make sure the documentation would compile, but not publish on pull requests, and only publish on push events.

We'll keep it commented out for now until I can resolve that test action.